client/src/Components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import { Form } from 'react-bootstrap';
+import { useHistory } from 'react-router';
+import NavbarComponent from '../Navbar/Navbar.js';
+
+const LoginPage = (props) => {
+	const history = useHistory();
+
+	async function handleClick (e) {
+		e.preventDefault();
+		const formElement = document.querySelector('form');
```

Comment:
I believe you can get the form instance as:
```suggestion
		const form = event.currentTarget;
```

Reference: https://react-bootstrap.github.io/components/forms/#forms-validation